enhancements/ingress/default-apps-domain.md

```diff
@@ -0,0 +1,118 @@
+---
+title: default-apps-domain
+authors:
+  - "@dustman9000"
+reviewers:
+  - "@Miciah"
+  - "@danehans"
+  - "@frobware"
+  - "@knobunc"
+  - "@sgreene570"
+approvers:
+  - "@knobunc"
+  - "@Miciah"
+creation-date: 2020-08-12
+last-updated: 2020-09-04
+status: implementable
+---
+
+# Default Apps Domain
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement allows for the configuration of a default apps domain in which
+can be used to set the default host and domain of the route, if not specified by
+the admin. Instead of using the default cluster domain, this new setting can be
+used to override it when set.
```

Comment:
Usually I assume "admin" means the cluster administrator, and this enhancement is focused on user-created routes, right?
```suggestion
This enhancement allows for the configuration of a default apps domain, which
can be used to specify an alternative default domain for user-created routes.
This alternative domain, if specified, overrides the default cluster domain for the
purpose of determining the default host for a newly created route.
```

Reply:
Good point. Yes, this is mainly for user-created routes.